recipes/chxtools/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "chxtools" %}
+{% set version = "3.0.2" %}
+{% set sha256 = "1d6bb5516c413f85f6502850b4e5563dc899d682adeb4aa242e1525ba59efce1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NSLS-II-CHX/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - databroker
+    - eiger-io
+    - matplotlib
```

Comment:
Unless you need a PyQt GUI, better to depend on matplotlib-base.

Reply:
@kbeeperez, could you please update the dependencies to use `matplotlib-base` instead of `matplotlib`? Thanks!

Reply:
Actually, there are some Qt-related imports, so we need to use `matplotlib`.

We may need this file to make the tests pass: https://github.com/nsls-ii-forge/chxtools-feedstock/blob/master/recipe/yum_requirements.txt.

Reply:
I don't see any here: https://github.com/NSLS-II-CHX/chxtools ?

Reply:
Oh, I did not look carefully at my traceback, which came from Matplotlib's `backend_qt5`. I think we can keep `matplotlib-base` as a lighter dependency. That way we won't need the `yum_requirements.txt` file.

@kbeeperez, could you please test that the suggested dependencies from https://github.com/nsls-ii-forge/chxtools-feedstock/pull/3 work with `matplotlib-base` here? Thank you.